c/src/CMakeLists.txt

```diff
@@ -31,7 +31,10 @@ set(libsbp_SRCS
   edc.c
   sbp.c
   )
-set(LIBSBP_CFLAGS "-fpic" CACHE STRING "Compile flags for libsbp.")
+
+if (NOT MSVC)
```

Comment:
so you could use this:
https://cmake.org/cmake/help/latest/prop_tgt/POSITION_INDEPENDENT_CODE.html

that should be more portable; I did this because @silverjam and @benjaminaltieri wanted to be able to turn this off easily.

Reply:
Thanks.